doc/source/whatsnew/v0.21.0.txt

```diff
@@ -39,6 +39,7 @@ Other Enhancements
 - :func:`read_feather` has gained the ``nthreads`` parameter for multi-threaded operations (:issue:`16359`)
```

Comment:
It got changed automatically by something as part of trying to build docs. I found it actually got deleted at some point and so reverted it from previous commits but the space was different for some reason.